src/library/scala/Unit.scala

```diff
@@ -28,6 +28,7 @@ final abstract class Unit private extends AnyVal {
 }
 
 object Unit extends AnyValCompanion {
+  def void[A](x: A): Unit = ()
```

Comment:
To a Java developer this might signal that it converts a Unit instance to a java.lang.Void.

Reply:
That's true for me. Should come up with another name.

Reply:
I agree.  I was reading https://github.com/rust-lang/rfcs/pull/1216#issuecomment-236557016 last night and realised this would introduce possible confusion.

Reply:
OTOH we could change the return type and do: `{ val c = classOf[java.lang.Void].getDeclaredConstructor(); c.setAccessible(true); c.newInstance() }` ;)

Reply:
You're right, Seb, I was getting my unit types and bottom types confused.

`Unit` and `java.lang.Void` are both unit types.  `java.lang.Void` is only inhabited by `null`, while `Unit` is only inhabited by `()` (and auto-boxes to `BoxedUnit` if upcast to `Any`.)

Still, it needn't necessarily be called `void`.  I'm torn between `ignore`, `discard` or `unused` and whether it should live on `Unit` or `Function`.

Reply:
Personally, I think defining it in `Predef` or `Function` both ok :)